contentpacks/khanacademy.py

```diff
@@ -356,11 +365,14 @@ def recurse_nodes(node, parent_path=""):
                             i += 1
 
         for child in children:
-            recurse_nodes(child, node.get("path"))
+            node_count += 1
```

Comment:
Oh, wait. We return the value of the deepest `node_count` it seems. So to to give the correct `node_count`:

seed: 0
plant: 1
orange: 2
orange juice: 3
dead plant: 4
fertilizer: 5

Is this correct?